econml/tests/test_federated_learning.py

```diff
@@ -74,9 +74,12 @@ def test_lineardrlearner(self):
 
             for cov_type in ['HC0', 'HC1', 'nonrobust']:
                 with self.subTest(n_t=n_t, cov_type=cov_type):
-                    est_all = LinearDRLearner(model_propensity=t_model, model_regression=y_model)
-                    est_h1 = LinearDRLearner(model_propensity=t_model, model_regression=y_model)
-                    est_h2 = LinearDRLearner(model_propensity=t_model, model_regression=y_model)
+                    est_all = LinearDRLearner(model_propensity=t_model, model_regression=y_model,
```

Comment:
Should we also test that aggregation is not possible when enable_federation=False?

Reply:
Good suggestion, will address in next revision.